group-coordinator/src/main/resources/common/message/ShareGroupPartitionMetadataValue.json

```diff
@@ -20,25 +20,21 @@
   "validVersions": "0",
   "flexibleVersions": "0+",
   "fields": [
-    { "name": "InitializedTopics", "versions": "0+", "type": "[]TopicPartitionsInfo",
-      "about": "The topics with initialized share-group state." },
-    { "name": "DeletingTopics", "versions": "0+", "type": "[]TopicInfo",
-      "about": "The topics whose share-group state is being deleted." }
-  ],
-  "commonStructs": [
-    { "name": "TopicPartitionsInfo", "versions": "0+", "fields": [
-      { "name": "TopicId", "type": "uuid", "versions": "0+",
-        "about": "The topic identifier." },
-      { "name": "TopicName", "type": "string", "versions": "0+",
+    { "name": "Topics", "versions": "0+", "type": "[]TopicMetadata",
```

Comment:
That's because the previous `ShareGroupPartitionMetadata` in the KIP has become the new `ShareGroupStatePartitionMetadata`. This is because the pattern is that every record in KIP-848 which has the name `ConsumerGroupXXXX` has an equivalent `ShareGroupXXXX`, and `ConsumerGroupPartitionMetadata` needed an equivalent called `ShareGroupPartitionMetadata`. Thus, I needed a new name for the previous `ShareGroupPartitionMetadata`, but the way it works has not changed. I will update the KIP shortly.